client/client-lib/src/ln/mod.rs

```diff
@@ -366,11 +370,15 @@ mod tests {
             Ok(TransactionStatus::Accepted {
                 epoch: 0,
                 outputs: vec![SerdeOutputOutcome::from(&OutputOutcome::from(
-                    mint.output_outcome(OutPoint {
-                        txid: tx,
-                        out_idx: 0,
-                    })
-                    .unwrap(),
+                    // FIXME: Need to block on this future because output_outcome is not send
```

Comment:
https://github.com/dpc/fedimint/commits/pr/1032

@m1sterc001guy Here are fixes for you to fix it.